Create vss-extension.json

MatthijsdeHeus opened the pull request:
Basic JSON file needed in order to publish to the Azure Marketplace.

janssen-io commented:
Het zou goed zijn om ook het volgende op te nemen:

- categorie
- content details + license
- repository

Zie deze referentie voor meer info: https://docs.microsoft.com/en-us/azure/devops/extend/develop/manifest?view=azure-devops

Zijn er geen scopes nodig?
https://docs.microsoft.com/en-us/azure/devops/extend/develop/manifest?view=azure-devops#scopes

Ook zie ik dat we nog geen license hebben. Misschien goed om gelijk een MIT license toe te voegen? (https://mit-license.org/)

MatthijsdeHeus commented:
I updated the manifest and added the MIT license.

The scope I added was the "vso.code_manage", this should allow the extension to do it's work. Maybe "vso.code_write" could also be an option?

janssen-io commented:
I think `vso.code_manage` is too much. We only pull and push to repository, so I think `vso.code_write` should be sufficient.

MatthijsdeHeus commented:
Alright, just changed it.
